corehq/blobs/tests/test_migrate.py

```diff
@@ -29,6 +29,7 @@
 class BaseMigrationTest(TestCase):
```

Comment:
```TestMigrateBackend``` would probably benefit too.  I just decided to stop at some arbitrary point for the day.  I'll make sure to get this one in the next batch.